cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1161,6 +1161,7 @@ data InstallFlags = InstallFlags {
     installUpgradeDeps      :: Flag Bool,
     installOnly             :: Flag Bool,
     installOnlyDeps         :: Flag Bool,
+    installIndexSnapshot    :: Flag Int,
```

Comment:
Yeah it really would be nicer if this used a more illuminating type like UTCTime.